apps/browser-extension-wallet/src/hooks/useWalletManager.ts

```diff
@@ -58,7 +58,12 @@ export interface CreateHardwareWallet {
   connectedDevice: Wallet.HardwareWallets;
 }
 
-type WalletManagerAddAccountProps = Omit<AddAccountProps<Wallet.AccountMetadata>, 'extendedAccountPublicKey'>;
+type WalletManagerAddAccountProps = {
+  walletId: string;
+  accountIndex: number;
+  metadata: AccountMetadata;
+  password?: string;
```

Comment:
This has an advantage for security: you can overwrite the bytes in the byte array, but can't force GC to dispose a string. By converting the string to a byte array as early as possible, you have a better chance of removing the secret from memory earlier.
```suggestion
  passphrase?: Uint8Array;
```

Reply:
done in 82a42907c63b1e8db4e81f2610257bd52a467436